README.md

```diff
@@ -37,8 +37,63 @@ to your Github account:
 
 You should now receive an email whenever the changelog is updated.
 
+### ?? ?? 2023
+
+The following changes were deployed into sandbox:
+
+#### cis-deductions-api
```

Comment:
Would it be possible to group all the `DYNAMIC` changes together for readability purposes?